shapely/linear.py

```diff
@@ -110,7 +114,17 @@ def line_merge(line, **kwargs):
     <MULTILINESTRING ((0 2, 0 10), (0 11, 5 10))>
     >>> line_merge(MultiLineString())
     <GEOMETRYCOLLECTION EMPTY>
+    >>> line_merge(MultiLineString([[(0, 0), (1, 0)], [(0, 0), (3, 0)]]), directed=True)
+    <MULTILINESTRING ((0 0, 1 0), (0 0, 3 0))>
```

Comment:
To illustrate the contrast between directed=True vs False, maybe you could show this exact example with the default behaviour as well (so that you can see the direction was changed to form a single line)